indico_jacow/client/MailingList.jsx

```diff
@@ -0,0 +1,168 @@
+// This file is part of the JACoW plugin.
+// Copyright (C) 2021 - 2024 CERN
+//
+// The CERN Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License; see
+// the LICENSE file for more details.
+
+// import getMailingListsURL from 'indico-url:plugin_jacow.mailing_lists';
+import mailingListSubscribeURL from 'indico-url:plugin_jacow.mailing_lists_subscribe';
+import mailingListUnsubscribeURL from 'indico-url:plugin_jacow.mailing_lists_unsubscribe';
+
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import ReactDOM from 'react-dom';
+import {Button, ListItem, Icon, ListContent, List, Checkbox} from 'semantic-ui-react';
+
+import {Translate} from 'indico/react/i18n';
+import {indicoAxios} from 'indico/utils/axios';
+
+import './MailingList.module.scss';
+
+export function MailingList({mailingLists}) {
+  const [subMailingLists, setSubMailingLists] = useState(
+    mailingLists.lists.filter(list => list.subscribed)
+  );
+  const [notSubMailingLists, setNotSubMailingLists] = useState(
+    mailingLists.lists.filter(list => !list.subscribed)
+  );
+  const [selectedSubscribed, setSelectedSubscribed] = useState([]);
+  const [selectedNotSubscribed, setSelectedNotSubscribed] = useState([]);
+
+  const subscribeList = async listsIds => {
+    try {
+      const response = await indicoAxios.post(mailingListSubscribeURL({lists_ids: listsIds}));
+      if (response.status === 200) {
```

Comment:
I don't think this is needed. Any error from the server should already throw an exception.

The common pattern we use for making requests that are not expected to fail (but show some feedback if they do) is this:

```js
try {
  await indicoAxios.post(...);
} catch (error) {
  handleAxiosError(error);
  return;
}
```